packages/babel-preset-stage-3/index.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   plugins: [
-    require("babel-plugin-transform-async-to-generator"),
-    require("babel-plugin-transform-exponentiation-operator")
```

Comment:
Should I revert it to I thought you want me to bump it =)
Ok reverted version back